extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-hive-metastore-service-chart/values.yaml

```diff
@@ -15,11 +15,16 @@ image:
   dockerRepo: "ghcr.io/"
 
 mysql:
+  commonLabels:
```

Comment:
A: Seems like we might want to be more precise and attach labels only to the resources that need it instead of using this `commonLables` field.